http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200520_kubernetes_refactor.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="jcanas" id="kubernetes_drop_refactor">
```

Comment:
Do these tables have any data in dev/alpha/perf/prod? Sometimes migrations work on an empty DB but fail when there is data -- want to make sure we avoid that here.

(Now that I'm typing this, we probably don't have any k8s records in dev/etc because the routes don't exist yet)